pkg/skaffold/runner/intent.go

```diff
@@ -39,6 +39,14 @@ func newIntents(autoBuild, autoSync, autoDeploy bool) *intents {
 	return i
 }
 
+func (i *intents) Reset() {
```

Comment:
how about `reset` since all the other methods are also unexported and the only use is in the current package.

Reply:
👍